recipes/leftbar/group_row/group_row.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <dt-recipe-general-row
+    :description="names"
+    :unread-count="unreadCount"
+    :selected="selected"
+    v-on="$listeners"
+  >
+    <template #left>
+      <dt-avatar
+        :initials="avatarInitials"
+        :group="groupCount"
+      >
+        <img
+          data-qa="dt-avatar-image"
+          :src="avatarSrc"
+          :alt="avatarInitials"
+        >
+      </dt-avatar>
+    </template>
+  </dt-recipe-general-row>
+</template>
+
+<script>
+import { DtRecipeGeneralRow } from '@/recipes/leftbar/general_row';
+import DtAvatar from '@/components/avatar/avatar.vue';
+
+export default {
+  name: 'DtRecipeGroupRow',
+
+  components: {
+    DtAvatar,
+    DtRecipeGeneralRow,
+  },
+
+  props: {
+    /**
+     * Optional avatar image url.
+     * if provided, it's also required to provide a value in the `avatarInitials` prop to use
+     * in the alt attribute of the avatar.
+     */
+    avatarSrc: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * Initials to display on the avatar if avatarSrc is not provided or
+     * alt attr if avatarSrc is provided.
+     */
+    avatarInitials: {
```

Comment:
Should this be required? Sounds like it should be based on this comment.

Reply:
Yeah makes sense, updated https://github.com/dialpad/dialtone-vue/pull/778/commits/392fdcb3d3ba9fd73853b38f494053e8194c569b